### ISSUE TITLE
Wrong format datetime on log message

### ISSUE THREAD

**Issue by hybner** [evryfs/github-actions-runner-operator#457]
I tried build and run latest version from master. We run it on Openshift 4. Log message has wrong format for datetime which is formmated as number:

```
I0712 14:11:34.980307 1 request.go:665] Waited for 1.046029539s due to client-side throttling, not priority and fairness, request: GET:https://100.98.0.1:443/apis/route.openshift.io/v1?timeout=32s
1.657635096634558e+09 INFO controller-runtime.metrics Metrics server is starting to listen {"addr": ":8080"}
1.6576350966348531e+09 INFO setup starting manager
1.6576350966351378e+09 INFO Starting server {"path": "/metrics", "kind": "metrics", "addr": "[::]:8080"}
1.6576350966351552e+09 INFO Starting server {"kind": "health probe", "addr": "[::]:8081"}
I0712 14:11:36.635221 1 leaderelection.go:248] attempting to acquire leader lease csas-github-actions-runner-operator/4ef9cd91.tietoevry.com...
I0712 14:11:54.074196 1 leaderelection.go:258] successfully acquired lease csas-github-actions-runner-operator/4ef9cd91.tietoevry.com
1.6576351140744388e+09 INFO controller.githubactionrunner Starting EventSource {"reconciler group": "garo.tietoevry.com", "reconciler kind": "GithubActionRunner", "source": "kind source: *v1alpha1.GithubActionRunner"}
1.6576351140744576e+09 INFO controller.githubactionrunner Starting EventSource {"reconciler group": "garo.tietoevry.com", "reconciler kind": "GithubActionRunner", "source": "kind source: *v1.Pod"}
1.6576351140744615e+09 INFO controller.githubactionrunner Starting EventSource {"reconciler group": "garo.tietoevry.com", "reconciler kind": "GithubActionRunner", "source": "kind source: *v1.Secret"}
1.657635114074464e+09 INFO controller.githubactionrunner Starting Controller {"reconciler group": "garo.tietoevry.com", "reconciler kind": "GithubActionRunner"}
1.6576351140743346e+09 DEBUG events Normal
....
```

**Comment by davidkarlsen** [comment]
I am not experiencing the same, my dates are on the format: `2022-07-27T20:43:44.273Z`
Do you set any special environment-variables?
Are you running the latest version?

**Comment by hybner** [comment]
I tried it again. I build new image from latest master (commit 6ed57086de4860020a24900fef5c60333a916a03) and it's same:
`I0803 07:17:45.351574 1 request.go:665] Waited for 1.046751239s due to client-side throttling, not priority and fairness, request: GET:https://100.98.0.1:443/apis/redhatcop.redhat.io/v1alpha1?timeout=32s
1.6595110670597703e+09 INFO controller-runtime.metrics Metrics server is starting to listen {"addr": ":8080"}
1.65951106706053e+09 INFO setup starting manager
1.6595110670607445e+09 INFO Starting server {"path": "/metrics", "kind": "metrics", "addr": "[::]:8080"}
1.6595110670607953e+09 INFO Starting server {"kind": "health probe", "addr": "[::]:8081"}
I0803 07:17:47.060830 1 leaderelection.go:248] attempting to acquire leader lease csas-github-actions-runner-operator/4ef9cd91.tietoevry.com...
I0803 07:18:04.125935 1 leaderelection.go:258] successfully acquired lease csas-github-actions-runner-operator/4ef9cd91.tietoevry.com
1.6595110841260054e+09 DEBUG events Normal {"object": {"kind":"ConfigMap","namespace":"csas-github-actions-runner-operator","name":"4ef9cd91.tietoevry.com","uid":"2698d346-f2f0-4ca8-b041-126a32b4fe57","apiVersion":"v1","resourceVersion":"166060145"}, "reason": "LeaderElection", "message": "github-actions-runner-operator-6457d7fd6c-hpwv9_f6e73995-8317-4508-91f7-5eca8cc9ef81 became leader"}
`
For run we use Openshift 4 - kubernetes v1.23.5+3afdacb.
We set these environments: WATCH_NAMESPACE, GITHUB_V3_API_URL, GITHUB_V4_API_URL, HTTP_PROXY, HTTPS_PROXY, NO_PROXY.
